Library/brew.rb

```diff
@@ -175,8 +175,8 @@ def require?(path)
 rescue UsageError
   onoe "Invalid usage"
   abort ARGV.usage
-rescue SystemExit
-  puts "Kernel.exit" if ARGV.verbose?
+rescue SystemExit => e
+  puts "Kernel.exit" if ARGV.verbose? && !e.success?
```

Comment:
Should we make that `$stderr.puts` instead? Doesn't seem helpful to mix this with regular output.